server/schedulers/hot_region.go

```diff
@@ -1095,18 +1207,22 @@ func (bs *balanceSolver) buildOperators() (ops []*operator.Operator, infl *stati
 
 	currentOp, typ, err := createOperator(bs.cur.region, srcStoreID, dstStoreID)
 	if err == nil {
-		bs.decorateOperator(currentOp, sourceLabel, targetLabel, typ, dim)
+		bs.decorateOperator(currentOp, false, sourceLabel, targetLabel, typ, dim)
 		ops = []*operator.Operator{currentOp}
-		infl = &statistics.Influence{
-			Loads: append(bs.cur.srcPeerStat.Loads[:0:0], bs.cur.srcPeerStat.Loads...),
-			Count: 1,
+		for _, revertRegion := range bs.cur.revertRegions {
+			currentOp, typ, err = createOperator(revertRegion, dstStoreID, srcStoreID)
+			if err != nil {
+				break
+			}
+			bs.decorateOperator(currentOp, true, targetLabel, sourceLabel, typ, dim)
+			ops = append(ops, currentOp)
```

Comment:
I wonder if the `RandBuckets` in`waiting_operator` will be influenced. The `GetOperator` may only get first one operator and miss the second one.